node/src/main/resources/swagger-ui/openapi.yaml

```diff
@@ -2275,6 +2474,9 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/AssetDistribution'
+            application/json;large-significand-format=string:
+              schema:
+                $ref: '#/components/schemas/AssetDistributionLSF'
```

Comment:
Response of `/assets/{assetId}/distribution` contains only map of assetId <-> balance